pkg/codegen/go/gen.go

```diff
@@ -45,6 +45,8 @@ import (
 // This token is always an invalid module since ':' is not allowed within modules.
 const ExternalModuleSig = ":always-external:"
 
+var notUtil = false
```

Comment:
Not sure how much I like having a global var for `false`. We'd generally just use an inline comment for this so you'd have `genHeader(..., false /* isUtil */)`

Reply:
huh! that works for me; felt a bit dirty 😂